src/main/java/com/Kh033Java/travelbook/entity/Note.java

```diff
@@ -17,29 +18,41 @@
     private Long id;
 
     private String title;
+
+    @JsonProperty
     private boolean isPublic;
+
     private String description;
-    @DateString("yyyy-MM-dd")
-    private Date dateOfVisiting;
+
+    @DateString("yy-MM-dd")
+    private Date date;
+
     private Integer peopleEstimate;
+
     private Integer pricesEstimate;
+
     private Integer cuisineEstimate;
+
     private Integer commonImpression;
 
+    @Relationship(type = "LIKED")
+    private User userThatLiked;
+    @Relationship(type = "CREATED_NOTE")
+    private User creatorUser;
```

Comment:
these relationships should be deleted only `@Relationship(type = "DESCRIBES")
    private City describedCity;	    private City describedCity;

    @Relationship(type = "HAS_PHOTO")	    @Relationship(type = "HAS_PHOTO")
    private Set<Photo> photos;	    private Set<Photo> photos;` is appropriate